tests/basic/extra_metrics_test.py

```diff
@@ -0,0 +1,31 @@
+from textwrap import dedent
+
+from tests.helpers.agent import Agent
+from tests.helpers.metadata import Metadata
+from tests.helpers.verify import verify
+from tests.monitors.expvar.expvar_test import run_expvar
+
+
+def test_extra_metrics_passthrough():
+    """
+    The specified extraMetrics should be allowed through even though they are
+    not included by default.
+    """
+    metadata = Metadata.from_package("expvar")
+
+    with run_expvar() as expvar_container_ip:
+        with Agent.run(
+            dedent(
```

Comment:
I've been removing `dedent` usage with seemingly no ill effects. Was also thinking of just adding it to `Agent.run` itself.

Reply:
Yeah I think as long as the YAML is otherwise indented properly the parser in the agent is tolerant of consistent leading whitespace.